components/outputs/data-grid/app-variation-select-rows-express.py

```diff
@@ -9,11 +9,11 @@
 
 @render.ui()
 def rows():
-    rows = input.penguins_df_selected_rows()  # <<
+    rows = penguins_df.input_cell_selection()["rows"]  # <<
```

Comment:
I think this is a good change in the sense of having the existing example run correctly with shiny 0.9.0.

We don't necessarily need to do it in this PR, but this example is less motivating now that the output object has more methods. It'd be nice if this example or a variant in this section gave more instruction about those methods and how to use them.